templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h2 class="subtitle">Delete Manufacturer</h2>
+  
+  <p>Are you sure you would like to delete this manufacturer {{ manufacturer }}?</p>
+  
+  <form action="" method="post">
+    {% csrf_token %}
+    <button class="btn btn-danger d-flex justify-content-center align-items-center addBtn" type="submit">

```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist